Frontify/arcade
# feat: add RichTextEditor

Add a draftjs rich text editor
Note: this is not the final one but it is already big and can be used to make blocks.

**Need to be merged:**
- https://github.com/Frontify/arcade/pull/67

**To test:**
1. `Frontify/arcade` => `feat/rich-text-editor` branch => `npm link <clarify path>/node_modules/react`
2. `Frontify/clarify` => `master` branch => `npm link @frontify/arcade`
3. `Frontify/guideline-blocks` => `task/text-block` branch => `npm run bootstrap`
4. `Frontify/guideline-blocks` => `task/text-block` branch => `npm link @frontify/arcade`
5. `Frontify/guideline-blocks` => `task/text-block` branch => `npx lerna run serve:block --scope=text-block --stream`
6. Go to a styleguide, use `Local Block Development` and you should have an empty text block.

A lot of steps are needed to avoid collisions between local React instances.

In case of `invalid hook call`, just ping me on Slack.
